tests/functional/api/BackbeatServer.js

```diff
@@ -458,11 +462,12 @@ describe('Backbeat Server', () => {
                         // We cannot guarantee order because it depends on how
                         // Redis fetches the keys.
                         keys.forEach(k => {
+                            // eslint-disable-next-line no-unused-vars
                             const [bucket, key, versionId, site] = k.split(':');
                             assert(res.Versions.some(o => (
                                 o.Bucket === bucket &&
                                 o.Key === key &&
-                                o.VersionId === versionId &&
+                                o.VersionId === testVersionId &&
```

Comment:
Because the response gets the version ID using the object MD instead of the key name (see change [here](https://github.com/scality/backbeat/pull/286/files/d4fa76fad604b5b98fffbe6760dd4a20da0eab34#diff-7012bb0401146916f9385c1c205fe3dcR243). I think it's better because it actually checks that the returned value is the correct version ID. I will go ahead an update the key names to use `testVersionId ` just to be consistent.